geopyspark/__init__.py

```diff
@@ -48,3 +49,78 @@ def create_python_rdd(pysc, jrdd, serializer):
         return RDD(jrdd, pysc, serializer)
     else:
         return RDD(jrdd, pysc, AutoBatchedSerializer(serializer))
+
+def geopyspark_conf(appName=None, master=None, additional_jar_dirs=[]):
```

Comment:
`master` and `appName` should be reversed in order to match the order of `SparkContext` and minimize errors from muscle memory.